src/components/Layout.tsx

```diff
@@ -0,0 +1,199 @@
+import Header from "@/components/Header";
+import NavBar from "@/components/NavBar";
+import TopLines from "@/components/TopLines";
+import Footer from "./homepage/footer";
+import React from "react";
+import PostLayout, {PostLayoutProps} from './news/PostLayout';
+import ShowcaseLayout, {ShowcaseLayoutProps} from './showcase/ShowcaseLayout';
+
+type Props = {
+  type?: 'news' | 'showcase' /* | 'guide' */;
+  news_props?: PostLayoutProps;
+  showcase_props?: ShowcaseLayoutProps;
+  /*guide_props?: GuideLayoutProps;*/
+  page_header?: string;
+  children?: React.ReactNode;
+};
+export default function Layout({ type, news_props, showcase_props, page_header, children }: Props) {
+  return (
+    <>
+      <Header title={"News"} />
```

Comment:
This should reflect the content on the page and not always be "News" for all Layouts.

Reply:
Oooh, good catch! thank you, I can quickly fix this one :pray:

Reply:
I've added a simple function with a `switch` statement here to render the title properly based on the provided `type`.

If we would prefer, I can write a quick function to split the type into words and capitalize the first letter of each word. I held off on this, since it didn't seem necessary, but if we would prefer this approach we can do that instead :)